workflows/diagnostics/fv3net/diagnostics/prognostic_run/load_run_data.py

```diff
@@ -82,7 +82,12 @@ def _coarsen_cell_centered_to_target_resolution(
 
 def _load_3d(url: str, catalog: intake.catalog.Catalog) -> xr.Dataset:
     logger.info(f"Processing 3d data from run directory at {url}")
-    files_3d = ["diags_3d.zarr", "state_after_timestep.zarr", "nudging_tendencies.zarr"]
+    files_3d = [
+        "diags_3d.zarr",
+        "state_after_timestep.zarr",
+        "nudging_tendencies.zarr",
+        "piggy.zarr",
```

Comment:
It should be fine if this file isn't present. load_coarse_data will return an empty dataset if the path isn't there.